"/**":
  ros__parameters:
    port: 554
    username: admin
    password: zjsxmfhf
    topic_name_prefix: /ipcam_
    transport: udp
    width: 1536
    height: 864
    publish_rate: 30.0
    use_compression: true
    jpeg_quality: 90

"ipcam_1":
  ros__parameters:
    ip: "192.168.0.27"
    camera_id: 1
    frame_id: "ipcam_1"
    K: [992.9704543278901, 0.0, 745.4080923633682, 0.0, 992.570868709759, 411.50030459149866, 0.0, 0.0, 1.0]
    new_K: [700.7187807174055, 0.0, 742.4814595173574, 0.0, 699.1715761698123, 410.1144091043476, 0.0, 0.0, 1.0]
    roi: [0, 0, 1536, 864]
    size: [1536, 864]
    dist: [-0.2748339201713551, 0.03801535894790896, 0.00011773843613683306, 0.0007567214551083291, 0.011612300134375477]
    
"ipcam_2":
  ros__parameters:
    ip: "192.168.0.21"
    camera_id: 2
    frame_id: "ipcam_2"
    K : [1039.627016222056, 0.0, 724.032103594604, 0.0, 1030.4126399697875, 379.9112839311773, 0.0, 0.0, 1.0]
    new_K : [714.0639257214872, 0.0, 714.3497270325074, 0.0, 698.82927683246, 392.23436267303174, 0.0, 0.0, 1.0]
    roi : [0, 0, 1536, 864]
    size : [1536, 864]
    dist : [-0.29279730177601715, 0.0430811875193046, 0.006354909392390921, 0.0026728851872214877, 0.008359780174727985]
    
"ipcam_3":
  ros__parameters:
    ip: "192.168.0.30"
    camera_id: 3
    frame_id: "ipcam_3"
    K: [1060.100434186973, 0.0, 776.0477686593179, 0.0, 1057.0681994621832, 382.11263751578167, 0.0, 0.0, 1.0]
    new_K: [764.686904340868, 0.0, 772.7377070283012, 0.0, 761.948806110288, 387.8990722293303, 0.0, 0.0, 1.0]
    roi: [0, 0, 1536, 864]
    size: [1536, 864]
    dist: [-0.3024705381825829, 0.032017760031894306, 0.0043627948354659435, -0.0012617235069007614, 0.027921356285954345]
    
"ipcam_4":
  ros__parameters:
    ip: "192.168.0.51"
    camera_id: 4
    frame_id: "ipcam_4"
    K: [1010.6851422748907, 0.0, 737.4630988619297, 0.0, 1008.4717560294529, 411.30662238930313, 0.0, 0.0, 1.0]
    new_K: [709.9527742950877, 0.0, 732.2844312192361, 0.0, 705.1203619510844, 409.34945145184236, 0.0, 0.0, 1.0]
    roi: [0, 0, 1536, 864]
    size: [1536, 864]
    dist: [-0.277031759468014, 0.03883138825843906, 0.00012012396336453722, 0.001378680949329653, 0.010211983400653218]

"ipcam_5":
  ros__parameters:
    ip: "192.168.0.25"
    camera_id: 5
    frame_id: "ipcam_5"
    K: [977.1439545790979, 0.0, 780.4020750240472, 0.0, 982.7008128620945, 434.11870859238576, 0.0, 0.0, 1.0]
    new_K: [665.4924478570485, 0.0, 762.4094750485358, 0.0, 678.366302958977, 400.9096438316427, 0.0, 0.0, 1.0]
    roi: [0, 0, 1536, 864]
    size: [1536, 864]
    dist: [-0.2608948805223571, 0.024812404537105388, -0.011624888437031569, -0.0038760594399617166, 0.01448773781714764]

"ipcam_6":
  ros__parameters:
    ip: "192.168.0.35"
    camera_id: 6
    frame_id: "ipcam_6"
    K: [1160.7928943519116, 0.0, 794.3521885660352, 0.0, 1162.6977337489295, 391.7899096719194, 0.0, 0.0, 1.0]
    new_K: [801.4698219378515, 0.0, 781.774123333379, 0.0, 804.0382092408474, 392.50234632550143, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.36392267240609, 0.066412742089692, 0.003617115195361055, -0.00490130321272115, 0.017893908455834925]

"ipcam_7":
  ros__parameters:
    ip: "192.168.0.50"
    camera_id: 7
    frame_id: "ipcam_7"
    K: [983.7286721431524, 0.0, 792.4878924039691, 0.0, 983.4121094029896, 407.2181011619916, 0.0, 0.0, 1.0]
    new_K: [671.1322663258572, 0.0, 787.2829027450306, 0.0, 677.3750876702804, 400.5221599210855, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.27216507530837397, 0.03930840220569434, -0.0010296604533974814, -0.002574906073060055, 0.007216662328203806]

"ipcam_8":
  ros__parameters:
    ip: "192.168.0.36"
    camera_id: 8
    frame_id: "ipcam_8"
    K: [1080.008610925944, 0.0, 779.441595151715, 0.0, 1076.4611340610697, 382.19498408518064, 0.0, 0.0, 1.0]
    new_K: [755.3707724094791, 0.0, 755.8810654059236, 0.0, 738.6560657909969, 392.05732829294317, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.3181797897944729, 0.053619445884378204, 0.006513949739114805, -0.005298113217218984, 0.012064814836195823]

"ipcam_9":
  ros__parameters:
    ip: "192.168.0.31"
    camera_id: 9
    frame_id: "ipcam_9"
    K: [966.3370117642802, 0.0, 783.1102404034674, 0.0, 969.9246580571148, 366.24780693212364, 0.0, 0.0, 1.0]
    new_K: [719.3349647647303, 0.0, 787.9808384091031, 0.0, 729.5249170154439, 382.68078832423134, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.2516325318199976, 0.0008977299707001173, 0.007505321246162587, 0.0008582074859144624, 0.03736411681547577]

"ipcam_10":
  ros__parameters:
    ip: "192.168.0.32"
    camera_id: 10
    frame_id: "ipcam_10"
    K: [1034.685198336495, 0.0, 740.181193331419, 0.0, 1026.5168237582852, 361.18137812206544, 0.0, 0.0, 1.0]
    new_K: [699.590713941937, 0.0, 741.7249424561444, 0.0, 697.7685096320009, 368.498022424135, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.3000882700638662, 0.05654237399706513, 0.006589811058608427, 0.002609065050166037, 0.002129386583769849]

"ipcam_11":
  ros__parameters:
    ip: "192.168.0.26"
    camera_id: 11
    frame_id: "ipcam_11"
    K: [988.1090072679679, 0.0, 752.8795629562067, 0.0, 986.530895154944, 411.7515526924019, 0.0, 0.0, 1.0]
    new_K: [704.7614055670471, 0.0, 771.5391567503041, 0.0, 696.4401066088839, 409.4443304714247, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.27086180443772456, 0.03703559370294543, -6.607409004123303e-05, 0.004440313178038263, 0.012542403122882172]

"ipcam_12":
  ros__parameters:
    ip: "192.168.0.37"
    camera_id: 12
    frame_id: "ipcam_12"
    K: [1038.877493884038, 0.0, 764.706727340762, 0.0, 1039.0930253350282, 412.1859413259871, 0.0, 0.0, 1.0]
    new_K: [749.7599269405044, 0.0, 766.9254419970811, 0.0, 749.8131442452901, 412.3068823457312, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.2839881614633001, 0.009750522722098191, 0.0005706742480525527, 0.0006186818983048614, 0.036336555761800976]

"ipcam_13":
  ros__parameters:
    ip: "192.168.0.29"
    camera_id: 13
    frame_id: "ipcam_13"
    K: [1046.83868456434, 0.0, 750.9035847155416, 0.0, 1046.6352632406392, 418.4721019491207, 0.0, 0.0, 1.0]
    new_K: [745.031221079087, 0.0, 742.5629595279396, 0.0, 741.9865648487205, 414.1984258099833, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.3156730127862902, 0.07983236722625911, -0.0009159842294278437, -0.00015422450151715282, -0.0004889617633316215]

"ipcam_14":
  ros__parameters:
    ip: "192.168.0.28"
    camera_id: 14
    frame_id: "ipcam_14"
    K: [999.1127329750218, 0.0, 754.7412899763762, 0.0, 998.028502612663, 429.58228345473054, 0.0, 0.0, 1.0]
    new_K: [715.6081913167031, 0.0, 747.5337679329425, 0.0, 709.2489862054978, 429.72472522304963, 0.0, 0.0, 1.0]
    roi:  [0, 0, 1536, 864]
    size: [1536, 864]
    dist:  [-0.26863438566097153, 0.025770753997803982, 0.00010896754992234254, -0.0006382335883252921, 0.018204007710752534]
